polyfill/index.d.ts

```diff
@@ -227,9 +227,29 @@ export namespace Temporal {
     milliseconds?: number;
     microseconds?: number;
     nanoseconds?: number;
+    /** @deprecated */ year?: number;
+    /** @deprecated */ month?: number;
+    /** @deprecated */ day?: number;
+    /** @deprecated */ hour?: number;
+    /** @deprecated */ minute?: number;
+    /** @deprecated */ second?: number;
+    /** @deprecated */ millisecond?: number;
+    /** @deprecated */ microsecond?: number;
+    /** @deprecated */ nanosecond?: number;
```

Comment:
I don't understand these changes. How does this help? What's the behavior with and without them?

Reply:
There is more context in [this comment](https://github.com/tc39/proposal-temporal/issues/325#issuecomment-690993637). It's so that TypeScript will not suggest those in autocomplete, but will not break when analyzing JS code that already uses them.

Reply:
On second thought, this was wrong, it should have been only the values and not the properties: #1005